pipelines/batch/src/main/java/com/google/fhir/analytics/ConvertResourceFn.java

```diff
@@ -133,20 +133,23 @@ public void writeResource(HapiRowDescriptor element)
         return;
       }
     }
-    totalParseTimeMillisMap.get(resourceType).inc(System.currentTimeMillis() - startTime);
+    if (totalParseTimeMillisMap.get(resourceType) != null)
```

Comment:
nit: consider wrapping these two lines into a function that gets the map, `resourceType`, and `startTime` as params, does the null check and then increment. Then call this function everywhere (here and below).